tests/integration/test_http_transport.py

```
@@ -64,7 +67,37 @@ def terminate_process_safely(process: subprocess.Popen, timeout: float = 5.0) ->
+            if self.path == "/api/v1/users/me":
+                body = json.dumps({"id": 1, "login": "test-integration", "name": "Test User"}).encode()
+                self.send_response(200)
+                self.send_header("Content-Type", "application/json")
+                self.send_header("Content-Length", str(len(body)))
+                self.end_headers()
+                self.wfile.write(body)
+            else:
+                self.send_response(404)
+                self.end_headers()
+
+        def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)